.cloud/.azure/aci_deploy.json

```diff
@@ -26,4 +26,13 @@
     "ssl_key_pem_file": "<your-aci-ssl-key-pem-file>",
     "ssl_cname": "<your-aci-ssl-cname>",
     "dns_name_label": "<your-aci-dns-name-label>"
+}
+
+  
+{
+    "name": "mywebservice",
+    "inference_source_directory": "src/deploy/",
+    "inference_entry_script": "score.py",
+    "conda_file": "environment.yml",
+    "test_enabled": true
```

Comment:
I think something went wrong here. Please remove the last part, if it is not required.